docs/CONTRIBUTING.rst

```diff
@@ -18,24 +18,72 @@ know like-minded people and make something great together.
-
-- Be kind and friendly.
-- Be sure to read the relevant part of the documentation before posting a question.
-- Ask questions in the `Forum <https://forum.lvgl.io/>`__ and use
-  `GitHub <https://github.com/lvgl/>`__ for development-related discussions.
-- Always fill out the post or issue templates in the Forum or GitHub (or at least provide equivalent information). It makes
-  understanding your contribution or issue easier and you will get a useful response faster.
-- If possible send an absolute minimal but buildable code example in order to reproduce the issue. Be sure it
-  contains all the required variable declarations, constants, and assets (images, fonts).
-- Use `Markdown <https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet>`__
-  to format your posts. You can learn it in 10 minutes.
-- Speak about one thing in one issue or topic. It makes your post easier to find later for
-  someone with the same question.
-- Give feedback and close the issue or mark the topic as solved if your question is answered.
-- For non-trivial fixes and features, it's better to open an issue first to discuss the
-  details instead of sending a pull request directly.
-- Please read and follow the Coding style guide.
+Ways to contribute
+-------------------
+
+- **Spread the Word**: Share your LVGL experience with friends or on social media to boost its visibility.
+- **Start the LVGL `repository <https://github.com/lvgl/lvgl>`__**
+- **Report a bug**: Open a `GitHub Issue <https://github.com/lvgl/lvgl/issues>`__ if something is not working.
+- **Join our `Forum <https://forum.lvgl.io/>`__**: Meet fellow developers and discuss quiestions
+- **Tell your ideas**: If you miss something from LVGL we would love to hear about it in a `GitHub Issue <https://github.com/lvgl/lvgl/issues>`__
+- **Develop features**: Develop a pitch! Learn more below.
+
+
+Feature development workflow
+----------------------------
+
+Terminology
+~~~~~~~~~~~
+
+- **Shaping**: Convert raw ideas into something that can be developed. It includes examining opportunities, finding similar feature requests, roughly planning how it shall be implemented, tested, and taught through docs and examples.
+- **Pitch**: A document that is the result of shaping.
+
+Schedule
+~~~~~~~~
+
+We are working in 8-week cycles, each cycle starting at the beginning of odd months (e.g, a cycle could be from March to April).
+
+- **Weeks 1-2: Cool Down** (A relaxed period to step back, plan, fix bugs, and make a release):
+  - Close the previous development phase: Feature stop, bug fixes, release, discuss anything that can be improved.
+  - Prepare for the next development phase: Ideas, shaping, pitch creation.
+- **Weeks 3-8: Develop** Work on the planned pitches.
+
+
+Feature Development Issue
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For each raw idea, there is a dedicated issue where it's being "shaped". Anyone can open an idea/feature request issue and anyone comment with ideas to help the shaping process.
```

Comment:
Correct grammar:
"and anyone **can** comment **new** ideas"

Reply:
Reworded.